test/plugins/wasi_logging/wasi_logging.cpp

```diff
@@ -18,9 +16,7 @@ namespace {
 
 WasmEdge::Runtime::Instance::ModuleInstance *createModule() {
   using namespace std::literals::string_view_literals;
-  WasmEdge::Plugin::Plugin::load(std::filesystem::u8path(
-      "../../../plugins/wasi_logging/" WASMEDGE_LIB_PREFIX
-      "wasmedgePluginWasiLogging" WASMEDGE_LIB_EXTENSION));
+  WasmEdge::Plugin::Plugin::loadFromDefaultPaths();
```

Comment:
We should add a comment to identify that `built-in plugin` is loaded in this function.